src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+function gallery(evt) {
+  evt.preventDefault();
+
+  if (evt.target.tagName === 'IMG') {
+    const link = evt.target.closest('a');
```

Comment:
Ensure that the `link` variable is not null before accessing its `href` property. If `evt.target` is not inside an anchor (`<a>`) element, `link` will be null, leading to a runtime error when trying to access `link.href`. Consider adding a null check for `link`.